masonite/view.py

```diff
@@ -51,3 +75,121 @@ def extend(self):
     def share(self, dictionary):
         self.dictionary.update(dictionary)
         return self
+
+    def cache_for(self, time=None, type=None):
+        """
+        Set time and type for cache
+        """
+
+        self.cache_time = time
+        self.cache_type = type
+
+        if self.__is_expired_cache():
+            self.__create_cache_template(self.template)
+        return self
+
+    # --------
+    # Privates
+    # --------
+
+    def __get_path_cache(self):
+        """
+        Get path location disk config
+        """
+
+        cache_config = self.container.make('CacheConfig')
+        path = cache_config.DRIVERS['disk']['location'] + "/"
+        return path
+
+    def __create_cache_template(self, template):
+        """
+        Save in the cache the template
+        """
+
+        self.container.make('Cache').store(
+            template + ":" + str(time.time()) + '.html',
+            self.rendered_template
+        )
+
+    def __cached_template_exists(self):
+        """
+        Check if the cache template exists
+        """
+
+        path = self.__get_path_cache()
+        find_template = glob.glob(path + "/" + self.template + ":*")
+        if find_template:
+            return True
+        return False
+
+    def __is_expired_cache(self):
+        """
+        Check if cache is expired
+        """
+
+        # If is forever
+        if self.cache_time is None and self.cache_time is None:
+            return False
```

Comment:
I think the problem here is that this is returning `False` meaning that the cache is not expired. If it is forever then it should be `True` because it should act as if the cache is expired. So making this return True could fix your bug